handbook/working_and_growing_here.md

```diff
@@ -262,7 +393,7 @@ In the past, dbt Labs would hire in any country except those we had already dete
 | United States  	| dbt Labs, Inc.             	| ADP              	| ADP  	| USD      	| Yes                   	|
 | Germany        	| dbt Labs GmbH              	| Overseas Connect 	| ADP  	| EUR      	|                       	|
```

Comment:
For sponsorship - the UK should be yes and Germany should be no

Reply:
And currency in UK is EUR; Germany is GBD? Looks like they were switched originally.

Reply:
Disregard, it was correct originally.